### DISCUSSION TITLE
Data precision in different query conditions.

### DISCUSSION THREAD

**Discussion by shygit-dev** [apache/skywalking#9041] [Q&A] [answered]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

UI (apache/skywalking-booster-ui)

### What happened

Why can multiple detailed data be displayed only within one hour when adjusting the time range; If you can only get three points in more than an hour, all the plots are strange.
![image](https://user-images.githubusercontent.com/59356131/167889095-75cb95f6-d162-4851-8372-786b58e95c4d.png)

### What you expected to happen

Where can I adjust this range? Or this is a bug.

### How to reproduce

Adjust the time range for more than 1 hour.

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment, marked as answer]
Metrics have 3 levels precision/dimensionality, minute, hour, and day. UI would choose the suitable precision/dimensionality automatically.
If you choose more than one hour, the precision would be `hour`, then when you choose more than one day, the precision would be `day`. This is what we called `downsampling`, as well as what it is called in the statistic science.
